src/api/v1/register.go

```diff
@@ -48,10 +48,17 @@ func registerWithToken(c echo.Context) error {
 	if err != nil {
 		return err
 	}
-	body.Discord = strings.TrimSpace(body.Discord)
-	if body.Token == "" || body.Discord == "" || body.Mcuuid == "" || body.Email == "" || body.Password == "" {
+	// TODO allow creating account without discord or minecraft
```

Comment:
from a user standpoint i completely disagree with this to be honest but its fine since its just a comment

Reply:
The idea is that you can _create_ an account without this but add it later through the dashboard. Meaning any specific 3rd party account is optional; you get the perks specific to whichever accounts you link.